cmake_modules/commonSetup.cmake

```diff
@@ -257,6 +260,7 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
   if ( CLIENTTOOLS_ONLY )
       set(PLATFORM OFF)
       set(DEVEL OFF)
+      set(USE_CASSANDRA OFF)
```

Comment:
I do not believe this is correct. Cassandra plugin should be in clienttools as it can be used when creating standalone executables.

Reply:
Is Cassandra somehow already omitted from a macOS client tools build?  Here is what I see in an installed 6.4.30/clienttools/plugins directory:

R.ecllib
couchbase.ecllib
java.ecllib
javascript.ecllib
kafka.ecllib
lib_eclblas.ecllib
lib_keylib.ecllib
lib_memcached.ecllib
lib_metaphone.ecllib
lib_redis.ecllib
lib_saltlib.ecllib
lib_thorlib.ecllib
lib_word.ecllib
libauditlib.dylib
libdebugservices.dylib
libdmetaphone.dylib
libfileservices.dylib
liblogging.dylib
libparselib.dylib
libstringlib.dylib
libtimelib.dylib
libunicodelib.dylib
libworkunitservices.dylib
mysql.ecllib
python.ecllib
python3.ecllib
sqlite3.ecllib
sqs.ecllib

If it is already omitted, then the the line you indicated should be wrapped to make it macOS-specific.